src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +54,13 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<People> selectMenByAge(List<People> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        Predicate<People> predictor = person -> person.getSex() == People.Sex.MAN
```

Comment:
```suggestion
        Predicate<People> filterMenByAge = person -> person.getSex() == People.Sex.MAN
```
let's have more informative names for all predicates (same for other predicates)

Reply:
Done